src/bitcoin-cli.cpp

```diff
@@ -867,6 +874,84 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+    std::string RESET, GREEN, BLUE, YELLOW, MAGENTA, CYAN;
+
+#ifndef WIN32
+    if (isatty(fileno(stdout))) {
+      // Only print colored text if OS is not WIN32 and stdout is connected to a terminal.
```

Comment:
Might be nice to have a way to force it on/off?

Reply:
Are you suggesting a `bitcon.conf` option or CLI argument to allow WIN32 users to print ANSI color code and non-WIN32 users to not print ANSI color code? For the former, I am planning on making a follow-up PR to allow that but I am not too sure if there is a strong use case for the latter tho.

Reply:
eg, most GNU tools support a `--color=always/never/auto`

Often useful when you want to pipe coloured output to `less` or such

Reply:
Good suggestion! Added `-color` option in https://github.com/bitcoin/bitcoin/commit/5a2a6b3bc5e2b1bd085f5a687f1992182b71ad62